css/navigation.css

```diff
@@ -72,4 +76,8 @@ li{
 
 a{
     text-decoration: none;
+}
+
+p:hover{
+   background-color: #ff0000;
```

Comment:
@Akpjunior94 This should be targeting the color of the text not the background
`color: #ff0000`

Reply:
okay boss... noted